### DISCUSSION TITLE
1.7.1

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#893] [Обновления]
## Список изменений

- Добавлена поддержка Coursetrain (coursetrain.net) (#706)
- Добавлена поддержка Ricktube (ricktube.ru)
- Добавлена поддержка Bilibili Bangumi (#852)
- Добавлена поддержка Incestflix (#705)
- Добавлена поддержка субтитров от сайта для Vimeo
- Включен обход Media CSP для player.vimeo.com
- Добавлена поддержка просмотра видео в плеере Яндекс Диска без перехода по публичной ссылка (Вы все еще должны открыть публичный доступ к файлу (Не к папке!!!)) (#837)
- Исправлено долгое ожидание перевода для новых запросов на перевод для YouTube (статус = 6) ([1#issuecomment-2433274910](https://github.com/ilyhalight/voice-over-translation/issues/1#issuecomment-2433274910), [868#issuecomment-2436080833](https://github.com/ilyhalight/voice-over-translation/issues/868#issuecomment-2436080833))
- Исправлена работа встраиваемого плеера Vimeo (player.vimeo.com), если видео залито в приватный доступ и доступно только через встраивание (#543, #828)
- Переработан новый аудиоплеер, который полностью работает на AudioContext

  По умолчанию данный плеер используется, только, чтобы обходить Media CSP на некоторыъ сайтах. Если вам нужен какой-либо из ниже перечисленных фиксов, то вам нужно будет снять галочку "Использовать только для обхода Media CSP"

  - Исправлена работа медиаклавиш, если включен новый плеер (#192, #841)
  - Исправлена работа PiP при активном переводе для браузера Arc (#357)
  - Опция "Обходить Media CSP" заменена на "Использовать только для обхода Media CSP" (речь про новый плеер), а так же, теперь, данная опция зависит от состояния опции "Использовать новый аудио плеер"

- Исправлена ошибка из-за которой озвучка иногда могла не стартовать со старым плеером (#840)
- Теперь, при преобразование секунд в минуты будет учитываться доля секунд, вычисляемая по формуле secs/60, и если доля секунд больше N%, то результат будет округляться до следующей минуты. Благодаря этому, будет меньше ситуаций, когда сервер вернул, что необходимо подождать, например 110 секунд, а расширение пишет, что осталось около 1 минуты
- Исправлено добавление обводки для слайдеров на некоторых сайтах
- Исправлено некорректное отображение чекбоксов на некоторых сайтах
- Переработана логика работы с аудио плеерами

## What's Changed
* Update to vot.js 1.3.0 by @ilyhalight in https://github.com/ilyhalight/voice-over-translation/pull/831
* Updated to vot.js 1.3.1 by @ilyhalight in https://github.com/ilyhalight/voice-over-translation/pull/832
* Fix incorrect checkbox name by @ilyhalight in https://github.com/ilyhalight/voice-over-translation/pull/836
* fix multimedia keys if browser support audio context by @ilyhalight in https://github.com/ilyhalight/voice-over-translation/pull/844
* Перенос Tone.js и исправление громкости by @SashaXser in https://github.com/ilyhalight/voice-over-translation/pull/862
* Rewrited audio player logic with Tone.js (temporary) by @ilyhalight in https://github.com/ilyhalight/voice-over-translation/pull/869
* Changed audio context player Tone.js --> chaimu by @ilyhalight in https://github.com/ilyhalight/voice-over-translation/pull/874
* Fix translation button on change video with old player by @ilyhalight in https://github.com/ilyhalight/voice-over-translation/pull/879
* Update 1.7.1 by @ilyhalight in https://github.com/ilyhalight/voice-over-translation/pull/892

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.7.0...1.7.1

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.7.1'>1.7.1</a>.</em>

**Comment by hrihoriart** [comment]
Ещё раз хочу выразить благодарность, просто лучшее расширение за последние несколько лет которое я видел

**Comment by R0man1ch** [comment]
в Safari на sequoia не срабатывает перевод видео, выдаёт ошибку
![Снимок экрана 2024-11-09 в 10 40 06](https://github.com/user-attachments/assets/c4099329-f15a-4b89-9f40-f4fea3ec4952)

как это решить?

**Reply by ilyhalight** [reply]
проверьте сейчас

**Comment by R0man1ch** [comment]
Спасибо🙌🏼
Сейчас всё работает 👏🏼👍🏼

Давно хотел избиваться от Яндекс браузера)) держал его только ради этой фичи

**Comment by ArtoriusN** [comment]
Не нашел в изменениях.
Сегодня включил и слышу что озвучивает другой голос, очень похожий на chatgpt.
Также сравнил с версией из Яндекс браузера, перевод тоже отличается.
Ушли на другую модель или Яндекс тестирует что-то?

**Reply by ilyhalight** [reply]
Это тестовые голоса из Яндекс браузера, которые короткое время там были включены. В 1.7.2 их можно выключить

**Reply by ArtoriusN** [reply]
Мне очень понравилось, не отключайте пожалуйста!
Появилась интонация, голос более человечный стал.

**Reply by ilyhalight** [reply]
> Мне очень понравилось, не отключайте пожалуйста

Не планировал удалять, пока доступны в апи Яндекса будут и в расширение

**Reply by R0man1ch** [reply]
> Это тестовые голоса из Яндекс браузера, которые короткое время там были включены. В 1.7.2 их можно выключить

Какая сейчас версия актуальна?

**Reply by ilyhalight** [reply]
> > Это тестовые голоса из Яндекс браузера, которые короткое время там были включены. В 1.7.2 их можно выключить
> 
> Какая сейчас версия актуальна?

1.7.2 - самая последняя

**Comment by R0man1ch** [comment]
а для IOS возможен какой-то подобный способ перевода видео?

**Reply by ilyhalight** [reply]
аналогично поставьте скрипт через какой-нибудь юзерскрипт менеджер

**Reply by R0man1ch** [reply]
> аналогично поставьте скрипт через какой-нибудь юзерскрипт менеджер

Не могу найти, который работает

Посоветуете какой-то ?

**Reply by ilyhalight** [reply]
> > аналогично поставьте скрипт через какой-нибудь юзерскрипт менеджер
> 
> Не могу найти, который работает
> 
> Посоветуете какой-то ?

попробуйте UserScripts (так и называется)
